### PR TITLE
Sidebar Navigation updated

### PR DESCRIPTION
**Pull request by Malekalarmash**
Updated sidebar navigation

### CODE REVIEW

**Review by brentonstrine — APPROVED** [review]
I noted one problem but it's not a blocker. You can go ahead and merge and fix that issue in the next PR!

File: public/pages/techtalk_bootcamp.html [hunk 1]
```diff
@@ -81,13 +81,14 @@ <h4 style="margin: 0; text-align: center;">Erin Shaw </h4>
       <aside class="aside-navigation small patch">
         <h4>Upcoming Talks</h4>
         <ul>
-          <li><a href='/pages/techtalk.html'>Feb. 16 - JP MugiziBuilding a Data Analytics Platform Using Google Cloud</a></li>
+            <li><ahref='/pages/techtalk.html' class="previous">Feb. 23 - Wesley Hall: Introduction to Digital Analytics </a></li>
```
Comment (brentonstrine) [inline public/pages/techtalk_bootcamp.html:84]: indentation should be 2 spaces deeper than the parent (the `<ul>` above)

File: public/pages/techtalk_no-code.html [hunk 2]
```diff
@@ -82,13 +82,14 @@ <h4 style="margin: 0; text-align: center;">Paul Beresuita </h4>
       <aside class="aside-navigation small patch">
         <h4>Upcoming Talks</h4>
         <ul>
-          <li><a href="/pages/techtalk.html">Feb. 9 - Erin Shaw: Are Code Bootcamps a fast-track to a career in tech?</a></li>
-          <li><a >Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>
+            <li><ahref='/pages/techtalk.html' class="previous">Feb. 23 - Wesley Hall: Introduction to Digital Analytics </a></li>
```
Comment (brentonstrine) [inline public/pages/techtalk_no-code.html:85]: indentation should be 2 spaces deeper than the parent (the `<ul>` above)

File: public/pages/techtalk_artificial-intelligence.html [hunk 3]
```diff
@@ -77,13 +76,14 @@ <h2>Tech Talk: Artificial Intelligence</h2>
       <aside class="aside-navigation small patch">
         <h4>Upcoming Talks</h4>
         <ul>
-          <li><a href="/pages/techtalk.html">Feb. 9 - Erin Shaw: Are Code Bootcamps a fast-track to a career in tech?</a></li>
-          <li><a >Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>
+            <li><ahref='/pages/techtalk.html' class="previous">Feb. 23 - Wesley Hall: Introduction to Digital Analytics </a></li>
```
[the remark this answers is not in the document]
Comment (Malekalarmash) [reply public/pages/techtalk_artificial-intelligence.html:79]: Heard!

**Pull request merged**